src/engine/node/mod.rs

```diff
@@ -378,6 +378,8 @@ impl Node {
         *self.m().color()
     }
 
-
+    pub fn children(&self) -> &Vec<Node> {
+        &(*self).children
```

Comment:
Out of curiosity: Is this really necessary? I would have thought that `&self.children` would work just as well.